aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -648,6 +656,10 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   TO_BATCHING_RULE("to.other", const Tensor&, bool, bool, optional<MemoryFormat>)
   m.impl("to.dtype_layout", to_dtype_layout_batching_rule);
 #undef TO_BATCHING_RULE
+#define UNARY_POINTWISE_VA(op, ...) m.impl(#op, \
+    unary_pointwise_batching_rule<Tensor (*)(const Tensor&, __VA_ARGS__), at::op, __VA_ARGS__>);
+  UNARY_POINTWISE_VA(clone, optional<MemoryFormat>);
```

Comment:
This adds support for the clone function as well?
If so should it be tested/mentioned in the commit msg?

Reply:
That was an accidental commit, let me remove that

Reply:
EDIT: Yes, I should be testing clone here.